tests/infra_test.go

```diff
@@ -308,7 +312,11 @@ var _ = Describe("[Serial]Infrastructure", func() {
 					Value:  "",
 					Effect: k8sv1.TaintEffectNoExecute,
 				})
-				_, err = virtClient.CoreV1().Nodes().Update(selectedNodeCopy)
+
+				err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
+					_, err := virtClient.CoreV1().Nodes().Update(selectedNodeCopy)
```

Comment:
The `Update` call here is not coming from the PR creator,  but just since I see it and for visibility: We can't use `Update` on k8s objects which we didn't create, when using the structured k8s client. We can only use `Patch` on such objects. If we don't follow that rule, we always risk removing data which we didn't want to remove.

Reply:
thanks Roman, will open an issue on it to tackle it 
please see also 
https://github.com/kubevirt/kubevirt/pull/4482
which improve a bit the RetryOnConflict, but still use Update

Reply:
https://github.com/kubevirt/kubevirt/issues/4485